mission_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "run_title": "SUAS Test Flight",
+    "run_description": "Test flight for SUAS 2025",
+    "real_mission_data_path": "flight/data/golf_data.json",
+    "sim_mission_data_path": "flight/data/golf_data.json",
+    "standard_object_count": 5,
```

Comment:
We will only have 4 objects this year. And I believe this will no longer be used once the standard object pipeline has been updated